packages/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@botpress/messaging-client",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Unrelated. This version is already released but I didn't commit the version change for some reason so this fixes it.